src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java

```diff
@@ -115,23 +115,24 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
         DynamicList dynamicList = caseData.getCicCase().getDraftOrderDynamicList();
         UUID code = dynamicList.getValue().getCode();
         String label = dynamicList.getValue().getLabel();
-        String[] dynamicListLabel = label.split(DOUBLE_HYPHEN);
```

Comment:
this part will be removed no need to merge

Reply:
Reverted